framework/mesh/logical_volume/rpp_logical_volume.h

```diff
@@ -8,6 +8,16 @@ namespace opensn
 /**Rectangular Parallel Piped (RPP) logical volume*/
 class RPPLogicalVolume : public LogicalVolume
 {
+  enum ConditionDirection
```

Comment:
Not a big fan of the name. How about something like `CornerName` since this is defining the corners of a RPP.

Reply:
Change made.